packages/common/src/store/cache/users/reducer.ts

```diff
@@ -64,13 +64,13 @@ const actionsMap = {
 }
 
 const buildInitialState = (ssrPageProps?: SsrPageProps) => {
-  // TODO: support user profile page. Only track page supported for now.
-
   // If we have preloaded data from the server, populate the initial
   // cache state with it
-  if (ssrPageProps?.track) {
+  if (ssrPageProps) {
+    const apiUser = ssrPageProps.user ?? ssrPageProps.track?.user ?? null
```

Comment:
interesting idea, id love for us to think how we go from SSRProps -> initial redux state generally. my guess is that each route is in charge of normalizing the data? this is chill for now however

Reply:
yea i wrote down a bullet point in my notes about this. I feel like each reducer setting up it's own state isnt terrible, but would be nice for it all to happen in one place and just be done

Reply:
Don't reducers by definition have to define their own initial state? I guess we could do something similar to redux persist to dispatch an action that sets the state

But I agree we can probably generalize/abstract this in the future to make it easier to use

Reply:
yeah i think it's cool to have each reducer set up it's own state, but doing stuff like `ssrPageProps.track.user` feels like it knows too much about ssr. Ideally reducers don't know much about that, just that it can receive an object of content it can use to init state?